.github/workflows/checks.yml

```diff
@@ -167,7 +163,7 @@ jobs:
           chmod u+x darwinia
           sudo mv darwinia /usr/bin
       - name: Launch darwinia
-        run: darwinia --chain koi-dev --tmp --alice --unsafe-force-node-key-generation --tracing-api debug,trace --frontier-backend-type sql &
+        run: darwinia --chain darwinia-dev --tmp --alice --unsafe-force-node-key-generation --tracing-api debug,trace --frontier-backend-type sql &
```

Comment:
I believe it should be Crab instead of Darwinia; we will use Crab as the testnet next.